### DISCUSSION TITLE
Issue with Patrol Flutter tests

### DISCUSSION THREAD

**Discussion by Vukan-Levi9** [leancodepl/patrol#2191] [Q&A]
I have a question regarding running tests in Flutter using Patrol framework. Sometimes when I run the test it passes and sometimes it just fails with this message in report.

EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK
The following message was thrown: Multiple exceptions (3) were detected during the running of the current test, and at least one was unexpected.

at pl.leancode.patrol.PatrolJUnitRunner.runDartTest(PatrolJUnitRunner.java:135) 
at care.incision.teams.MainActivityTest.runDartTest(MainActivityTest.java:31)

patrol: ^3.6.1
Patrol CLI version: 2.7.0
Flutter 3.19.6

**Comment by adrian-moisa** [comment]
I think this one should be a bug report ticket. You are far more likely to get an answer from the team. As for the error, haven't seen it so far.
